packages/bpk-component-navigation-stack/readme.md

```diff
@@ -0,0 +1,53 @@
+# bpk-component-navigation-stack
+
+> Backpack navigation stack component.
+
+## Installation
+
+```sh
+npm install bpk-component-navigation-stack --save-dev
+```
+
+## Usage
+
+```js
+import React from 'react';
+import { updateOnDirectionChange } from 'bpk-component-rtl-toggle';
+import BpkNavigationStack from 'bpk-component-navigation-stack';
+
+const RtlAwareNavigationStack = updateOnDirectionChange(BpkNavigationStack);
```

Comment:
I'd remove the RTL aware code from the readme. This is mostly there for our storybooks so that we don;t need to reload the page.